css/css-variables/variable-cssText.html

```diff
@@ -31,8 +31,8 @@
         { element: "target1",   expectedCssText: "--var: var1;" },
         { element: "target2",   expectedCssText: "margin: var(--prop);" },
         { element: "target3",   expectedCssText: "background: var(--prop);" },
-        { element: "target4",   expectedCssText: "margin: var(--prop)  !important;" },
-        { element: "target5",   expectedCssText: "background: var(--prop)  !important;" },
+        { element: "target4",   expectedCssText: "margin: var(--prop) !important;" },
+        { element: "target5",   expectedCssText: "background: var(--prop) !important;" },
```

Comment:
I _think_ this test change is right, but it's a bit subtle / it's just because of how the whitespace-trimming stuff should work in custom properties. Because otherwise the space should be preserved in those.

cc @andruud, @tabatkins in case you recall the spec better than I do :)